src/renderer/stores/chat.ts

```diff
@@ -364,6 +373,82 @@ class ChatStore extends Store<ChatStoreState> {
       runtime.updateBadge()
       saveLastChatId(chatId)
     },
+    jumpToMessage: async (msgId: number) => {
+      // these methods were called in backend before
+      // might be an issue if DeltaBackend.call has a significant delay
+      const _message = await DeltaBackend.call('messageList.getMessages', [msgId])
+
+      const message = _message[msgId] as MessageType
+      if (message === null || message === undefined) {
+        throw new Error('jumpToMessage: Tried to jump to non existing message with id: ' + msgId)
+      }
+
+      const chatId = (message as MessageType).chatId
+
+      const chat = <FullChat>(
+        await DeltaBackend.call('chatList.selectChat', chatId)
```

Comment:
only needed if its not the current chat

Reply:
No we always do it. This way we start clean every time you jump to a message.

Reply:
If that makes things easier for a first step.